homeassistant/components/logbook/__init__.py

```diff
@@ -733,14 +746,12 @@ def data_domain(self):
     @property
     def attributes(self):
         """State attributes."""
-        if not self._attributes:
-            if (
-                self._row.attributes is None
-                or self._row.attributes == EMPTY_JSON_OBJECT
-            ):
+        if self._attributes is None:
+            source = self._row.shared_attrs or self._row.attributes
+            if source == EMPTY_JSON_OBJECT or source is None:
                 self._attributes = {}
             else:
-                self._attributes = json.loads(self._row.attributes)
+                self._attributes = json.loads(source) or {}
```

Comment:
When can `json.loads` return a falsey value if you already detect empty JSON object and `None`?

Reply:
I don't think it can anymore now that the guard above has been added